python/lsst/analysis/tools/analysisPlots/analysisPlots.py

```diff
@@ -75,11 +78,24 @@ def setDefaults(self):
             vectorKey="{band}_psfFluxErr", selector=VectorSelector(vectorKey="starSelector")
         )
 
+        self.process.filterActions.patch = DownselectVector(
+            vectorKey="patchWhole", selector=VectorSelector(vectorKey="starSelector")
+        )
+
+        self.process.calculateActions.stars = ScatterPlotStatsAction(
```

Comment:
You need to import `ScatterPlotStatsAction`. This is probably a rebase issue, as the import was removed on `main` recently.